index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>review-own-code</title>
+</head>
+
+<body>
+    <div id="main">
+        <h1 id="title">Dr.Norman Borlaug</h1>
+        <h1>The man who saved a billion lives</h1>
+        <div id="img-div">
+            <img id="image" src="./images/dr-norman.jpeg" alt="">
+            <div id="img-caption">
+                <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields
+                    - part
+                    of his life-long war on hunger.</p>
+            </div>
+        </div>
+
+        <div id="tribute-info">
+            <h5>Here's a time line of Dr. Borlaug's life:
+            </h5>
+            <ul>
+                <li>
+                    <p><strong>1914</strong> - Born in Cresco, Iowa</p>
+                </li>
+                <!-- <li>
+                    <p><strong>1923</strong> - went to LACAS School</p>
+                </li> -->
```

Comment:
Nice of you to encourage the student for their effort :+1:. When requesting a change, you should give an explanation as to why it was requested, so you don't leave the student in a state of confusion. it's possible the change isn't as apparent as you think. It would be best if you also made suggestions instead of giving orders to sound friendlier :smile:.